### ISSUE TITLE
E-mail test dont dysplay fishing alert

### ISSUE THREAD

**Issue by condealisson** [Zimbra-Community/spoofing-and-phishing-alert-zimlet#6]
Hi guys, congratulations for yours projects.

I installed and activated this package but it not show the alert with the imported test e-mail.
Have another way to test the zimlet?
My Zimbra is the ZCS-8.8.9_GA_3019.

**Comment by barrydegraaff** [comment]
did you deploy in this folder: /opt/zimbra/zimlets-deployed/_dev/tk_barrydegraaff_sa_alert ??

**Comment by condealisson** [comment]
Yes, I tryed deploy in zimlets-deployed/_dev/tk_barrydegraaff_sa_alert and normal deploy installing by zip in the zimlets/, the zimlet appears for the users in both cases, but no sucess with the e-mail test.

**Comment by adriangibanelbtactic** [comment]
I think this issue should be reopened.
Same problem on 8.8.11.GA.3737 .
Why is spamassassin to blame here in the first place?
I also imported the test email and I thought that:
```
SA_AlertZimlet.prototype.onMsgView
```
would be able to read your test message searching for: `URI_PHISH` and then popup the warning window.

I cannot test this issue in a former ZCS version right now but this seems a regression for me.

**Comment by adriangibanelbtactic** [comment]
Yes, I confirm this is a bug because some variables being empty situation has not been taken into account.

Here there is my fix: https://github.com/btactic/spoofing-and-phishing-alert-zimlet/commit/731ef7a63f8322311a165ba69f28dacc5e3d0b5e and an additional one: https://github.com/btactic/spoofing-and-phishing-alert-zimlet/commit/71795725ad47dbd36886d9ef51b0ae7ade859df3 .

I might ask for a pull request later when I finish my overall improvements.

**Comment by barrydegraaff** [comment]
great! 

Kind regards, 

Barry de Graaff 
Zeta Alliance 
Co-founder & Developer 
zetalliance.org | github.com/Zimbra-Community 

Signal: +31 617 220 227 
Fingerprint: 97f4694a1d9aedad012533db725ddd156d36a2d0
